app/javascript/components/ResultsPanel.js

```diff
@@ -1,51 +1,25 @@
 import React, { useContext } from 'react'
-import Tab from 'react-bootstrap/lib/Tab'
-import Tabs from 'react-bootstrap/lib/Tabs'
 import { StudySearchContext } from 'components/search/StudySearchProvider'
-
+import StudyResults from 'components/StudyResults'
+/**
+ * Component for Results displayed on the homepage
+ */
 const ResultsPanel = props => {
   const searchContext = useContext(StudySearchContext)
-  return (
-    <div id="results-panel">
-      {searchContext.results.studies && <StudyResults results={searchContext.results}/>}
-    </div>
-  )
-}
-
-const StudyResults = props => {
-  let displayedResults
-  if (props.results.studies.length>0) {
-    displayedResults = props.results.studies.map(result => (
-      <div key={result.accession} className='card'>
-        <Study
-          study={result}
-          handleStudyLabel = {props.handleStudyLabel}
-        />
-      </div>
-    ),
-    )
-  } else {
-    displayedResults = <p>No Results</p>
+  let displayedResults = <span></span>
+  if (searchContext.isLoaded) {
+    if (searchContext.results.studies.length>0) {
+      displayedResults = <StudyResults
+        results={searchContext.results}
+        handlePageTurn={pageNum => {searchContext.updateSearch({ page: pageNum })}}
+      />
+    } else {
+      displayedResults = <p>No Results</p>
+    }
   }
-
   return (
-    <Tab.Content id ='results-content'>
+    <div id="results-panel">
```

Comment:
this is great, I like how you did all the logic beforehand in pure javascript so the template is easy to read